api/consensus.go

```diff
@@ -6,6 +6,8 @@ import (
 
 	"github.com/NebulousLabs/Sia/types"
 
+	"fmt"
+	"github.com/NebulousLabs/Sia/modules"
```

Comment:
ahhh sry I know, but I keep forgetting to double check after the IDE adds an import -.-